generators/app/index.js

```diff
@@ -34,24 +35,50 @@ module.exports = (props) => {
     exec('git', ['clone', repo], {}, function () {
       // compile template
       const projectPath = path.join(process.cwd(), props.templateRepo.split('/').pop())
-      generateApp(projectPath, destinationPath, props)
-      exec('rm', ['-rf', projectPath])
+      // check if choo.yaml exist
```

Comment:
The process here to line 64 seems a bit complex, it might be better to break these out into smaller bits.  I haven't tested it though, but if its working I suppose this could be fine for 1.0